app/connectors/ApmConnector.scala

```diff
@@ -70,13 +64,14 @@ class ApmConnector @Inject() (http: HttpClient, config: ApmConnector.Config, met
   def fetchAllAPICategories()(implicit  hc: HeaderCarrier): Future[Seq[APICategoryDetails]] =
     http.GET[Seq[APICategoryDetails]](s"${config.serviceBaseUrl}/api-categories")
 
-  def fetchAPIDefinition(serviceName: String)(implicit hc: HeaderCarrier): Future[ExtendedApiDefinition] =
-    http.GET[ExtendedApiDefinition](s"${config.serviceBaseUrl}/combined-api-definitions/$serviceName")
+  def fetchAPIDefinition(serviceName: String)(implicit hc: HeaderCarrier): Future[ApiDefinition] =
```

Comment:
combined-api-definitions is meant to return extended api definitions.  If you want just api definitions you need another endpoint I would have though.

Reply:
Yes it does - And for clarity it may be best to bring it back. I think it was removed as a convenience.
I'll sort it.